docs/building-modules.md

```diff
@@ -174,3 +241,7 @@ Tracking issue: n/a
 #### Other
 
 - `XMLHttpRequest` Standard
+
+```
+
+```
```

Comment:
WIP?

Reply:
I think that's VS Code "format on save" in action 🙈 Thanks for catching this!

Reply:
Fixed in 82f36a5